swagger.yaml

```diff
@@ -246,6 +246,55 @@ paths:
           description: Wrong content type (should be plain text)
         422:
           description: Missing fields
+  /projects/{projectId}/collaboration:
+    put:
+      tags:
+        - projects
+        - collaboration
+      summary: Creates an invite token for project sharing
+      security:
+        - user_id: []
+      requestBody:
+        conent:
+          application/json:
+            schema:
+              type: object
```

Comment:
Should there be any body?

Reply:
No, this is intentional. 
This endpoint (PUT) adds an entry (project_id,  NULL, token) to a database and returns that token (the third value in this tuple).
POST endpoint takes the token in body and looks for the matching project_id, token_id and fills the nullified value with user_id, the request sender. 
I decided to use PUT instead of GET for the first endpoint, since it creates some new entity rather than retrieving already existing data.

Reply:
That's not what I meant here. You should use some schema of the request object or omit requestBody section. Requesting body with arbitrary object is probably not fine.